### ISSUE TITLE
Cleanup on Inactive

### ISSUE THREAD

**Issue by rozzingit** [Arx-Game/arxcode#260]
**Is your feature request related to a problem? Please describe.**
So there's a few cleanup items that happen when a character is rostered, and I don't believe any of them happen when a character is set inactive. Two that have stood out to me are 1) Voices aren't deranked, and 2) patronage/protege relationships aren't broken. Voice cleanup is really helpful 

**Describe the solution you'd like**
I think it'd be helpful to include automatic things that happen upon setting a character Inactive the way it does when you set them as Available. De-Voicing people at Rank 2 and breaking patronage relationships would be really good to do. (A patron shouldn't be codedly benefiting from an Inactive protege.)

**Comment by Zironic** [comment]
There are some things that are disabled on being inactive such as the fact a crafter can no longer score critical hits in their shop but I do agree with you Roz that it would make a lot of sense for things like patrons and voice relationships being cleaned up on being marked inactive.

**Comment by rozzingit** [comment]
Yeah, but stuff like that is checked when people are running commands, versus the roster cleanup items which make the most sense to happen when the switch of character status changes.
